spec/support/boxes/issue_box.cr

```diff
@@ -0,0 +1,14 @@
+class IssueBox < BaseBox
+  def initialize
+    status Issue::AvramStatus.new(0)
```

Comment:
Does this allow you to do `status Issue::AvramStatus::Opened`?

Reply:
Sadly no. The macro create two things. an enum called `Status`, and a class that works as a wrapper as `AvramStatus` we use as a type. Even Issue::Status would be best here it won't work. And AvramStatus is not an Enum, so ::Opened won't work.

It will work to send `:opened` instead of 0 though.